src/components/AssetsFilters/AssetsFilters.test.jsx

```diff
@@ -39,5 +41,56 @@ describe('<AssetsFilters />', () => {
       checkBoxes = checkBoxGroup.find('[type="checkbox"]');
       expect(checkBoxes.first().html()).toContain('checked');
     });
+    it('correctly maps updateFilter to dispatch props', () => {
+      const dispatchSpy = jest.fn();
+
+      const { updateFilter } = mapDispatchToProps(dispatchSpy);
+
+      const updateFilterAction = {
+        data: {
+          Code: 'Code',
+        },
+        type: assetActions.filter.FILTER_UPDATED,
+      };
+
+      updateFilter('Code', 'Code');
+
+      expect(dispatchSpy).toHaveBeenCalledTimes(1);
+      expect(dispatchSpy).toHaveBeenCalledWith(updateFilterAction);
+    });
+    it('correctly maps updatePage to dispatch props', () => {
+      const dispatchSpy = jest.fn();
+
+      const { updatePage } = mapDispatchToProps(dispatchSpy);
+
+      const updatePageAction = {
+        data: {
+          page: 0,
+        },
+        type: assetActions.paginate.PAGE_UPDATE,
+      };
+
+      updatePage(0);
+
+      expect(dispatchSpy).toHaveBeenCalledTimes(1);
+      expect(dispatchSpy).toHaveBeenCalledWith(updatePageAction);
+    });
+    it('calls updatePage & updateFilter when filter box is checked', () => {
+      const filterSpy = jest.fn();
+      const pageUpdateSpy = jest.fn();
+
+      wrapper.setProps({
+        updateFilter: filterSpy,
+        updatePage: pageUpdateSpy,
+      });
+
+      const checkBoxGroup = wrapper.find('CheckBoxGroup');
+      const checkBoxes = checkBoxGroup.find('[type="checkbox"]');
+      const checkBox = checkBoxes.first();
+      checkBox.simulate('change', { target: { checked: true, type: 'checkbox' } });
+
+      expect(pageUpdateSpy).toHaveBeenCalledTimes(1);
```

Comment:
The two tests look good. The only thing I can think of is also adding something like:

`expect(pageUpdateSpy).toHaveBeenCalledWith(0)` to make sure that we're actually resetting the page back to 0 when a filter is clicked. Does that seem reasonable?

I'd suggest the same thing for filterSpy, but I'm not sure what the filter is off the top of my head. I'd imagine something like: 

*click filter for Images*
*expect filterSpy to be called with { Images: true }, or whatever it is that it's called with*

Reply:
That does make sense and sounds reasonable -- thanks!